### ISSUE TITLE
Makefile VERSION is set to 0.3.4 in tag 0.3.5

### ISSUE THREAD

**Issue by SweBarre** [kube-vip/kube-vip#216]
**Describe the bug**
The VERSION variable in the Makefile for tag 0.3.5 is still set to 0.3.4
I think that 's why my manifest output with image 0.3.5 creates a manifest that is still using 0.3.4 image
**To Reproduce**
using containerd
1. `crictl pull docker.io/plndr/kube-vip:v0.3.5`
2. 
```
ctr run --rm --net-host docker.io/plndr/kube-vip:v0.3.5 \
        vip /kube-vip manifest daemonset \
          --arp \
          --interface eth0 \
          --vip 10.2.0.250 \
          --controlplane \
          --leaderElection \
          --inCluster \
          --taint
```
3. see output for the container image tag
```
apiVersion: apps/v1
kind: DaemonSet
metadata:
  creationTimestamp: null
  name: kube-vip-ds
  namespace: kube-system
spec:
  .... truncated....
  template:
    .... truncated....
    spec:
      containers:
        - args:
          .... truncated....
          image: plndr/kube-vip:0.3.4
          .... truncated....    
```
**Expected behavior**
I expected that the manifest created should be using image v0.3.5 tag and not 0.3.4

**Environment (please complete the following information):**
 - OS/Distro: openSUSE Leap 15.3
 - Kubernetes Version: v1.20.7+k3s1
 - Kube-vip Version: 0.3.5

**`Kube-vip.yaml`:**
```
apiVersion: apps/v1
kind: DaemonSet
metadata:
  creationTimestamp: null
  name: kube-vip-ds
  namespace: kube-system
spec:
  selector:
    matchLabels:
      name: kube-vip-ds
  template:
    metadata:
      creationTimestamp: null
      labels:
        name: kube-vip-ds
    spec:
      containers:
      - args:
        - manager
        env:
        - name: vip_arp
          value: "true"
        - name: vip_interface
          value: eth0
        - name: port
          value: "6443"
        - name: vip_cidr
          value: "32"
        - name: cp_enable
          value: "true"
        - name: cp_namespace
          value: kube-system
        - name: vip_ddns
          value: "false"
        - name: vip_leaderelection
          value: "true"
        - name: vip_leaseduration
          value: "5"
        - name: vip_renewdeadline
          value: "3"
        - name: vip_retryperiod
          value: "1"
        - name: vip_address
          value: 10.2.0.250
        image: plndr/kube-vip:0.3.4
        imagePullPolicy: Always
        name: kube-vip
        resources: {}
        securityContext:
          capabilities:
            add:
            - NET_ADMIN
            - NET_RAW
            - SYS_TIME
      hostNetwork: true
      nodeSelector:
        node-role.kubernetes.io/master: "true"
      serviceAccountName: kube-vip
      tolerations:
      - effect: NoSchedule
        key: node-role.kubernetes.io/master
        operator: Exists
  updateStrategy: {}
status:
  currentNumberScheduled: 0
  desiredNumberScheduled: 0
  numberMisscheduled: 0
  numberReady: 0
```

**Comment by davidjaquier** [comment]
Agree! this issue is really annoying

**Comment by thebsdbox** [comment]
Fixed, and rebuild 0.3.5 with the fix.
